app/bundles/CampaignBundle/Executioner/Event/ActionExecutioner.php

```diff
@@ -60,7 +60,6 @@ public function execute(AbstractEventAccessor $config, ArrayCollection $logs)
         // Execute to process the batch of contacts
         $pendingEvent = $this->dispatcher->dispatchEvent($config, $event, $logs);
 
-        /** @var ArrayCollection $contacts */
         $passed = $this->eventLogger->extractContactsFromLogs($pendingEvent->getSuccessful());
         $failed = $this->eventLogger->extractContactsFromLogs($pendingEvent->getFailures());
```

Comment:
Types are known from the `extractContactsFromLogs()` return type strict declaration :+1: